### DISCUSSION TITLE
Updated OpenID Connect image?

### DISCUSSION THREAD

**Discussion by lhoekenga** [NginxProxyManager/nginx-proxy-manager#2417] [General]
Has anyone brought the openidc branch up to date?

I've got 2.9.19 running, but I'd really like to use oidc for authn. I forked the repo and made a [new openidc branch](https://github.com/lhoekenga/nginx-proxy-manager/tree/openidc), bringing it in sync with "develop", but I haven't been able to build the docker image.

The merge only had one conflict which was easy to fix, but I have no idea if my branch works.

thx

Liam

**Comment by JeWe37** [comment]
I've run into the same issue. In general documenting the build process would really be helpful for anyone looking to contribute.
